internal/common/version.go

```diff
@@ -36,24 +38,35 @@ func BaseVersionGreaterOrEqual(version, versionMayGreaterThan string) (bool, err
 }
 
 func BaseVersionLessThan(version, versionMayLessThan string) (bool, error) {
-	isGraterOrEqual, err := BaseVersionGreaterOrEqual(version, versionMayLessThan)
+	isGreaterOrEqual, err := BaseVersionGreaterOrEqual(version, versionMayLessThan)
 	if err != nil {
 		return false, err
 	}
-	return !isGraterOrEqual, nil
+	return !isGreaterOrEqual, nil
```

Comment:
just nit, make it more readable - doesn't make much differents 

```suggestion
	isGreaterOrEqual, err := BaseVersionGreaterOrEqual(version, versionMayLessThan)
	if err != nil {
		return false, err
	}
	if isGreaterOrEqual { 
	         return false, nil
	}
	return true, nil
```